Sources/Widgets/Widgets3D/LineWidget/index.js

```diff
@@ -62,6 +60,12 @@ function vtkLineWidget(publicAPI, model) {
 
   function initializeHandleRepresentations() {
     HandleRepresentation[0] = shapeToRepresentation[model.handle1Shape];
```

Comment:
```HandleRepresentation[0] = shapeToRepresentation[model.handle1Shape] || vtkSphereHandleRepresentation;```

Reply:
HandleRepresentation is a constant, do not modify it.
Create a mapping function instead of a variable ?

Reply:
I can understand the point of not modifying constant variable. What i don't understand is the point of using a mapping function instead.  Plus i think i messed it up by putting HandleRepresentation in constants. I think adding the handleRepresentation tab in the scope of this object would be better.

Reply:
This was linked to the state tree